File: BACK/Traffic Police/model/OwnerDTO.go
package model

import (
	"encoding/json"
	"io"
)

type OwnerDTO struct {
	Jmbg      string `json:"jmbg"`
	FirstName string `json:"firstName"`
	LastName  string `json:"lastName"`
	Email     string `json:"email"`
}

type Owners []*OwnerDTO

func (o *Owners) ToJSON(w io.Writer) error {
	e := json.NewEncoder(w)
	return e.Encode(o)
}

func (o *OwnerDTO) ToJSON(w io.Writer) error {
	e := json.NewEncoder(w)
	return e.Encode(o)
}

func (o *OwnerDTO) FromJSON(r io.Reader) error {
	d := json.NewDecoder(r)
	return d.Decode(o)
}

func (o *Owners) FromJSON(r io.Reader) error {
	d := json.NewDecoder(r)
	return d.Decode(o)
}

File: BACK/traffic-police/src/main/java/com/example/traffic_police/model/OwnerDTO.java
package com.example.traffic_police.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OwnerDTO {
    private String id;
    private String firstName;
    private String lastName;
    private String address;
    private String jmbg;
    private String email;
}
